x-pack/test_serverless/api_integration/test_suites/common/index_management/inference_endpoints.ts

```diff
@@ -67,7 +66,7 @@ export default function ({ getService }: FtrProviderContext) {
         inferenceEndpoints.some(
           (endpoint: InferenceAPIConfigResponse) => endpoint.inference_id === inferenceId
         )
-      ).to.be(true);
+      ).to.eql(true, `${inferenceId} not found in the GET _inference/_all response`);
```

Comment:
I don't think we should change these tests to expect the error response

Reply:
Introduced to improve indication on what caused the test cases to fail